src/freertos_drivers/st/Stm32Can.hxx

```diff
@@ -91,11 +95,15 @@ private:
     static unsigned int intCount;
 
     uint8_t state_; ///< present bus state
-
-    /** Default constructor.
-     */
+    
+    CAN_TypeDef *can_; ///< CAN hardware registers for this instance (defined in the SDK).
+    IRQn_Type can_irqn_; ///< CAN IRQn for this instance.
```

Comment:
These variable names do not follow the style guide (https://github.com/bakerstu/openmrn/blob/master/doc/style_guide.dox#L363). They should be camelCase, like
`canIrqn_` or `canIrqN_` or `canIRQn_` at your discretion.
Also below `canSecondIrqn_` etc. You could also consider `canInterrupt_` and `canSecondInterrupt_` if the acronyms are confusing for capitalization.

Reply:
Fixed.